src/Bridges/bridge_optimizer.jl

```diff
@@ -632,8 +632,8 @@ function get_all_including_bridged(
 )
     list = MOI.get(b.model, attr)
     if !isempty(Variable.bridges(b))
-        # TODO(odow): Fix this!
-        list = append!(copy(list), keys(Variable.bridges(b)))
+        # The conversion from `keys` into a `Vector` happens inside `append!`.
+        append!(list, keys(Variable.bridges(b)))
```

Comment:
This might modify the `list`. Is it clearly stated in `get` that what is returned can be modified ?

Reply:
This does modify `list`. The `get` docstring doesn't mention it, but anything else is just asking for trouble.

Reply:
The `get_all_including_bridged` function below also modifies `list`, so any model violating this principle is already wrong.